packages/falter-berlin-network-defaults/uci-defaults/freifunk-berlin-pingcheck-defaults

```diff
@@ -11,5 +11,7 @@ uci set pingcheck.@default\[0\].host=livecheck.berlin.freifunk.net
 # set the default timeout to 60 seconds
 uci set pingcheck.@default\[0\].timeout=60
 
-uci commit pingcheck
+# activate panic.d scripts
+uci set pingcheck.@default\[0\].panic='1'
 
+uci commit pingcheck
```

Comment:
So here we have a case where this will not be migrated to.  So anyone with an old setup will not get this change.  I have a suggestion though.  How about creating a file uci-defaults/freifunk-berlin-pingcheck-berlin-2020-07-30 with the set and commit.  This would possibly be a better way of dealing with migrations from now on.  As you get a new package version, then you also get one or more new uci-defaults files which (albetically) get ran.  Update complete and related to the package.

Reply:
I agree on that. Migration via uci-defaults-scripts sound pretty neat to me.